apps/aehttp/test/aehttp_contracts_SUITE.erl

```diff
@@ -1146,21 +1146,21 @@ remote_gas_test_contract(Config) ->
     call_get(APub, APriv, EncC2Pub, Contract, 100),
     force_fun_calls(Node),
     Balance1 = get_balance(APub),
-    ?assertMatchVM(1600476, 1600010, (Balance0 - Balance1) div ?DEFAULT_GAS_PRICE),
+    ?assertMatchVM(1600596, 1600010, (Balance0 - Balance1) div ?DEFAULT_GAS_PRICE),
```

Comment:
I need to understand why these numbers changed for Sophia if the PR is for FATE only. 🔬

Reply:
I added functions to the contract, and the way AEVM works is that the code layout affects the gas cost because of the function dispatch in the beginning of each contract,

While I didn't verify that this corresponds exactly to these changed numbers, the numbers changed are in the correct range.

Reply:
Yes, I see contract `remote_gas_test` changed.